ynput/ayon-core
# Chore: Move TimersManager addon

## Changelog Description
Moved timers manager client code next to server codebase.

## Additional information
The addon cares about automated stop of timer after inactivity, and unifies timers across multiple addons.

## Testing notes:
1. Create package of timers manager and core.
2. Upload both to server.
3. Make sure AYON launcher is using client code from this PR (e.g using dev paths).
4. Timers manager integration should work as before.
5. One example is that e.g. ftrack timer will start when application is launched.
